<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setiing</class>
 <widget class="QDialog" name="Setiing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="n1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MSL2-Python设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_download_auto_backup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>下载自动备份插件（需要服务端支持插件）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dl_update">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>检查MSL2更新</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_update">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>65</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>Master</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monthly</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dev</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="n2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>更新通道:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="n3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Master用于接收最新的稳定版本&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Monthly用于接收较稳定的Beta更新，新加入的功能可能会因为bug用不了，如果您新发现了Bug请前往Github提交issues&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Dev用于接收最新开发版，但Bug不是一般的多，如果您不是想协助我修复Bug请不要选择此通道(推荐先issues讨论然后再pull request)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_goto_website">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <item>
    <property name="text">
     <string>Forge官网</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fabric官网</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Cbukkit官网(下载)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spigot官网(下载)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vanilla官网(下载)</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pbtn_goto">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>前往</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="update_log">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="n4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>更新日志</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_check_configs_for_server">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>检查服务端路径下所有配置文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_download_website_manage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>211</width>
     <height>22</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>下载网页管理插件（需要服务端支持插件）</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>以下插件我觉得特别有用但都不是我写的:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_ufl1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>常用插件列表1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_ufl2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>常用插件列表2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
